<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoterRecords</class>
 <widget class="QDialog" name="VoterRecords">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1399</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2980b9;</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1399</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid white;
                    padding: 0 8px;
                    background-color: green;
                    font: 500 11pt ;
                   

}

 </string>
   </property>
   <property name="text">
    <string> LBTCSU VOTING  SYSTEM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Voter_back">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>50</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::GoHome"/>
   </property>
  </widget>
  <widget class="QPushButton" name="StudentRegistration_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>641</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">     
QPushButton{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 6px;
                    padding: 0 8px;
                    background-color: green;
                    font: 500 11pt ;
                   

}

 </string>
   </property>
   <property name="text">
    <string>SEARCH VOTER RECORD</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>1241</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
    background-color: rgb(0, 54, 79); /* Your header color */
    border-top-left-radius: 0px;
    border-top-right-radius: 0px;
    border-bottom-left-radius: 12px;
    border-bottom-right-radius: 12px;
    color: white;
}
QTableView {
    background-color: #1B263B;
    color: #FFFFFF;
    selection-background-color: #1B98E0;
    gridline-color: #415A77;
    border-top-left-radius: 12px;
    border-top-right-radius: 12px;
    border-bottom-left-radius: 12px;
    border-bottom-right-radius: 12px;
}
QPushButton {
    background-color: #1B98E0;
    color: white;
    border-radius: 6px;
    padding: 6px 12px;
}
QLineEdit {
    background-color: #2E3A59;
    color: white;
	border-radius: 7px;
	border: 1 solid blue;
}
QComboBox {
	background-color: #2E3A59;
    color: white;
	border-radius: 7px;
	border: 1 solid blue;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_search_voter">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Search_Voter">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:2px solid  green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Enter Search Query</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_voter_view">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
    </property>
    <property name="text">
     <string>View All Records</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>631</width>
      <height>411</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>StudentID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fname</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Lname</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Department</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ImageData</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ImageName</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_Reset_Voter">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>140</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ViewRestore"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
